packages/jest-preset-mc-app/setup-test-framework.js

```diff
@@ -2,6 +2,7 @@ require('unfetch/polyfill');
 require('jest-enzyme');
 require('jest-dom/extend-expect');
 require('react-testing-library/cleanup-after-each');
+const IntlPolyfill = require('intl');
```

Comment:
This took me some time to figure out, as it was only "broken" in test environment 🤬